queue/redis/redis.go

```diff
@@ -152,7 +153,7 @@ func pingQueue(client *redis.Client) error {
 		return nil
 	}
 
-	return fmt.Errorf("unable to establish Redis queue connection")
+	return fmt.Errorf("unable to establish Redis queue connection - %s", err)
```

Comment:
Can we use `%w` in this one? I think that would work to help propagate the error up the stack